index.js

```diff
@@ -1,7 +1,7 @@
 module.exports = visit;
 
 function visit(old) {
-    if (typeof(old) !== 'object' && old !== null) {
+    if (typeof old !== 'object' || old === null) {
```

Comment:
@revolunet why are you combining a functional change with a style change? I like (perhaps liked) to do `typeof(old)` in here.

Respect the style of the repo you're contributing to.